src/fu-util.c

```diff
@@ -1658,6 +1941,24 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Install prepared updates now"),
 		     fu_util_install_prepared);
+	fu_util_add (priv->cmd_array,
+		     "history",
+		     NULL,
+		     /* TRANSLATORS: command description */
+		     _("Show history of firmware updates"),
+		     fu_util_history);
+	fu_util_add (priv->cmd_array,
+		     "history-clear",
```

Comment:
Similarly this should be like "clear-results" and "clear-history"

Reply:
Fixed.